de.allotropia.ZetaOffice.json

```diff
@@ -0,0 +1,147 @@
+                "/lib/pkgconfig",
+                "/var"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz",
+                    "sha256": "b7a4cd5ead67fb08b980b21abd150ff7217e85ea320c9ed0c6dadd304840ad35",
+                    "x-checker-data": {
+                        "type": "html",
+                        "url": "https://kerberos.org/dist/",
+                        "version-pattern": "Kerberos V5 Release ([\\d\\.]+) - current release",
+                        "url-template": "https://kerberos.org/dist/krb5/$major.$minor/krb5-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "zetaoffice",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.zetaoffice.net/flathub/ZetaOffice-24-2_143_artifacts_zeta-flatpak-files.tar.xz",
+                    "sha512": "52e095cec80b938c8a1b7ac8475ec1e2b6d0323c11e30c0db27fa89d302cb6836eef7b6ad0adfc03de22455963f57f9c946f337e2c781b7f634340d07e94ec98",
+                    "dest": "source",
+                    "strip-components": 0
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/bin && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/bin/zetaoffice /app/bin/",
+                "mkdir -p /app/share && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/share/app-info /app/share/",
+                "mkdir -p /app/share && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/share/metainfo /app/share/",
+                "mkdir -p /app/share/applications && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/share/applications/* /app/share/applications/",
+                "mkdir -p /app/share/icons && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/share/icons/* /app/share/icons/",
+                "mkdir -p /app/share/runtime/locale && cp -a source/runtime/de.allotropia.ZetaOffice.Locale/x86_64/master/active/files/{*,.ref} /app/share/runtime/locale/",
+                "mkdir -p /app && cp -a source/app/de.allotropia.ZetaOffice/x86_64/master/active/files/zetaoffice /app/",
+                "mkdir -p /app/zetaoffice/help && cp -a source/runtime/de.allotropia.ZetaOffice.Help/x86_64/master/active/files/{*,.ref} /app/zetaoffice/help/"
+            ]
+        }
+    ],
+    "add-extensions": {
+        "de.allotropia.ZetaOffice.BundledExtension": {
+            "directory": "zetaoffice/share/extensions",
+            "subdirectories": true,
+            "no-autodownload": true,
+            "autodelete": true
+        },
+        "de.allotropia.ZetaOffice.Help": {
+            "directory": "zetaoffice/help",
+            "bundle": true,
```

Comment:
needs exception

Reply:
see <https://github.com/flathub-infra/flatpak-builder-lint/pull/636> "Exceptions for new de.allotropia.ZetaOffice"